### PR TITLE
Add driver labels

### PR DESCRIPTION
**Pull request by adrianchiris**
Add first feature implementation to expose MOFED version feature label

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,4 +7,10 @@
 [![CodeQL](https://github.com/Mellanox/nic-feature-discovery/actions/workflows/codeql.yml/badge.svg)](https://github.com/Mellanox/nic-feature-discovery/actions/workflows/codeql.yml)
 [![Image push](https://github.com/Mellanox/nic-feature-discovery/actions/workflows/image-push-main.yml/badge.svg?event=push)](https://github.com/Mellanox/nic-feature-discovery/actions/workflows/image-push-main.yml)
 
-NVIDIA NIC feature discovery discovers NIC related features to be advertised by Node Feature Discovery
+NVIDIA NIC feature discovery discovers NIC related features to be advertised by Node Feature Discovery.
```
Comment (adrianchiris) [inline README.md:10]: This is a bit thin... i will update the  README with more content.

File: pkg/daemon/daemon.go [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+			break OUTER
+		case <-time.After(d.scanInterval):
+			d.discover(ctx)
+		}
+	}
+}
+
+func (d *Daemon) discover(ctx context.Context) {
+	log := logr.FromContextOrDiscard(ctx)
+	log.WithName("discovery-daemon")
```
Comment (moshe010) [inline pkg/daemon/daemon.go:65]: maybe nic-discovery-daemon?

File: pkg/label/label.go
Comment (ykulazhenkov) [inline pkg/label/label.go:1]: Can `k8s.io/apimachinery/pkg/labels` be used instead of this package?

File: pkg/daemon/daemon.go (hunk 2 above)
Comment (adrianchiris) [reply pkg/daemon/daemon.go:65]: sure will rename

File: pkg/writer/writer.go [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+	return lr.writeLabels(labels)
+}
+
+// getCurrentLabels returns current labels set in  feature file
+func (lr *labelWriter) getCurrentLabels() ([]label.Label, error) {
+	curLabels := make([]label.Label, 0)
+
+	_, err := deps.OS.Lstat(lr.featureFilePath)
+	if err != nil {
+		if os.IsNotExist(err) {
```
[the remark this answers is not in the document]
Comment (adrianchiris) [reply pkg/writer/writer.go:89]: lemme validate this one and make needed modifications if applicable.

[the remark this answers is not in the document]
Comment (adrianchiris) [reply pkg/writer/writer.go:89]: done

**Review by ykulazhenkov — APPROVED** [review]

**Pull request merged**